Assets/TiOKawa/Scenes/MinaTest/Scripts/SoundView.cs

```diff
@@ -0,0 +1,36 @@
+using TiOKawa.Scripts.View;
+using UnityEngine;
+
+namespace TiOKawa.Scenes.MinaTest.Scripts.View {
+    public class SoundView : MonoView
```

Comment:
レビュー漏れててごめんなさい！
この命名だとBGMなのかSEなのかはたまた両方含んでいるのかがわからないため、名前の変更お願いしたいです
こんな感じで
```suggestion
    public class SEPlayerView : MonoView
```